fission-cli/library/Fission/CLI/Handler/App/Publish.hs

```diff
@@ -76,25 +72,30 @@ publish watchFlag runner appURL appPath _updateDNS updateData = do -- FIXME upda
       absBuildPath <- liftIO $ makeAbsolute buildDir
       logDebug $ "Starting single IPFS add locally of " <> displayShow absBuildPath
 
-      CLI.IPFS.Add.dir absBuildPath >>= putErrOr \cid@(CID hash) -> do
```

Comment:
I also expanded this out, because out little `>>= putOrErr` is just... ugly I think?

Reply:
We could also turn some of these inline `CLI.Error.put err; raise err` into showing the message next to the exit code (now that it's all set up like that). It would make it less noisy for the common case. Thoughts?